src/daria/image/image.py

```diff
@@ -217,6 +219,12 @@ def timestamp(self) -> datetime.datetime:
     def original_dtype(self) -> np.dtype:
         return self.metadata["original_dtype"]
 
+    # ! ---- Corresponding setter functions for metadata
+
+    @timestamp.setter
```

Comment:
never seen this @ commant before. does it work s one would imagine, i.e.,:
#initialize image 
img = da.Image(...)
#set time (where time is in the correct format of course)
img.timestamp = time

Reply:
This is exactly what this allows. Without the setter method (provided the extra @ command), it is not allowed to access a defined @property.